README.md

```diff
@@ -42,7 +49,7 @@ cargo run
 To sync a wallet to the default electrum server:
 
 ```shell
-cargo run -- wallet --descriptor "wpkh(tpubEBr4i6yk5nf5DAaJpsi9N2pPYBeJ7fZ5Z9rmN4977iYLCGco1VyjB9tvvuvYtfZzjD5A8igzgw3HeWeeKFmanHYqksqZXYXGsw5zjnj7KM9/*)" sync
+cargo run --features electrum -- wallet --descriptor "wpkh(tpubEBr4i6yk5nf5DAaJpsi9N2pPYBeJ7fZ5Z9rmN4977iYLCGco1VyjB9tvvuvYtfZzjD5A8igzgw3HeWeeKFmanHYqksqZXYXGsw5zjnj7KM9/*)" sync
```

Comment:
Few lines ago we mentioned how to install `bdk-cli` with `cargo`. 

Instructing to use `cargo run` here will build the local files instead of using the installed binary. So if someone is in their cloned repo, using cargo run they will actually build and then run the master branch instead of the release binary he just installed (this might be an unexpected behavior for the user, unless he knows cargo stuffs). 

Even worse if someone didn't clone the repo, `cargo run` will not run anything, even if they have `bdk-cli` binary installed (and we don't have that instruction anywhere).   

I think in the  "bdk-cli bin usage examples" section, we should just instruct to use `bdk-cli` binary instead of `crago run`, which will make the instruction simpler also.

This doesn't have to be fixed here. Just mentioned as it occurred to me.

Reply:
I agree we should probably update the README to focus on installing with different features, and then usage with an installed version instead of using `cargo run` since anyone making local changes should already know how to use cargo. But let's do that cleanup in a different PR.